Add message info box component

MostafaMohamed2002 opened the pull request:
**PR Description:**

This PR introduces a new reusable UI component, the "message info box". This component is designed to display a message with a title, a caption, and two action buttons.

**Changes:**

*   Added a new layout file `message_info_box.xml` for the component.
*   Added a new drawable `message_info_box_logout_background.xml` for the background of the icon in the message box.
*   Added a new icon `logout.png` to be used in the message box.

**Data Binding for Reusability:**

The `message_info_box.xml` layout uses data binding to make it highly reusable. The following variables can be set from the code:

*   `title`: The title of the message box.
*   `caption`: The caption of the message box.
*   `disabledButton`: The text for the disabled button.
*   `enabledButton`: The text for the enabled button.
*   `onDisabledClick`: The click listener for the disabled button.
*   `onEnabledClick`: The click listener for the enabled button.

This allows the component to be easily configured for different scenarios, such as logout confirmation, displaying error messages, or showing informational dialogs.

This component can be used in various parts of the application to display important messages to the user, such as a confirmation for logging out.

## screenshots
<img width="418" height="278" alt="image" src="https://github.com/user-attachments/assets/4de8b64c-d7b7-4ae8-b156-be840bb35b01" />

MostafaMohamed2002 committed:
Add logout icon

MostafaMohamed2002 committed:
Add background drawable for logout message info box

MostafaMohamed2002 committed:
Add new layout file `message_info_box.xml`

This layout defines a UI component for displaying informational messages with a title, caption, and two buttons.

MuhammedWael9991 commented on app/src/main/res/drawable/logout.png:
I suggest using ic_logout_duetone from resources instead of using png photo.

MostafaMohamed2002 replied:
Thanks , i don't know why android studio decides to import it as png 😕

MostafaMohamed2002 committed:
Replace logout icon in message info box

The `logout.png` drawable has been replaced with `ic_logout_duetone`.
Additionally, the ImageView displaying the icon in `message_info_box.xml` now has its tint set to `@color/brand_primary`.
